packages/app-content-pages/package.json

```diff
@@ -49,6 +50,7 @@
   },
   "devDependencies": {
     "@babel/register": "~7.18.6",
+    "@next/bundle-analyzer": "~13.0.4",
```

Comment:
This will break production apps, since they run `yarn install --production` and `next start` will fail if the bundle analyser isn’t installed.